README.md

```diff
@@ -46,6 +46,7 @@ Approvers ([@open-telemetry/dotnet-approvers](https://github.com/orgs/open-telem
 - [Bruno Garcia](https://github.com/bruno-garcia), Sentry
 - [Christoph Neumueller](https://github.com/discostu105), Dynatrace
 - [Liudmila Molkova](https://github.com/lmolkova), Microsoft
+- [Mikel Blanchard](https://github.com/CodeBlanch)

```

Comment:
This has been discussed during the 6/2, 2020 SIG meeting.

@CodeBlanch would you confirm which company shall we put here?

Reply:
@reyang Please put CoStar Group on there. Sorry for the delay, I wanted to get approval from my boss.

Reply:
Sure. And no worry - we're not going to merge this PR until we get your explicit approval 😄.
Thanks for all your contributions btw!